tests/unit/components/playback/controls/InputDropDown.spec.js

```diff
@@ -64,7 +64,7 @@ describe("InputDropDown.vue", () => {
       "Select Customer ID"
     );
   });
-  test("When the InputDropDown is mounted, Then the user enters few charters in the input, confirm that an event 'update' is emitted with entered text", async () => {
+  test("Given that the user types, When a valid option from the list is entered, Then confirm that an event 'update' is emitted with entered text", async () => {
```

Comment:
a "Given" section is not necessary in every test title. If there is nothing special about the way you are setting up the environment or mounting the component, then it is fine to just start with a "When" section.   

In this test, the user typing is the direct action that triggers the change we are trying to test. For example, something like: `When the user types a valid option from the list, Then an event 'update' is emitted with the entered text`